.gitlab-ci-check-golang-unittests.yml

```diff
@@ -93,3 +93,42 @@ publish:unittests:
         fi;
         sleep 1;
       done
+
+publish:unittests:
+  stage: publish
+  except:
+    - /^saas-[a-zA-Z0-9.]+$/
+  image: golang:1.14-alpine3.11
+  dependencies:
+    - test:unit
+  before_script:
+    - apk add --no-cache git
+    # Run go get out of the repo to not modify go.mod
+    - cd / && go get github.com/mattn/goveralls && cd -
+    # Coveralls env variables:
+    #  According to https://docs.coveralls.io/supported-ci-services
+    #  we should set CI_NAME, CI_BUILD_NUMBER, etc. But according
+    #  to goveralls source code (https://github.com/mattn/goveralls)
+    #  many of these are not supported. Set only CI_PR_NUMBER and
+    #  pass few others are command line arguments.
+    #  See also https://docs.coveralls.io/api-reference
+    - export CI_PR_NUMBER=${CI_COMMIT_BRANCH#pr_}
+  script:
+    - tar -xvf unit-coverage.tar
+    - goveralls
+      -repotoken ${COVERALLS_TOKEN}
+      -service gitlab-ci
+      -jobid $CI_PIPELINE_ID
+      -covermode set
+      -flagname unittests
+      -parallel
+      -coverprofile $(find tests/unit-coverage -name 'coverage.txt' | tr '\n' ',' | sed 's/,$//')
```

Comment:
I notice that `tr` has become a bit of a fan favourite lately! :p

Reply:
I use it a lot. But mainly because of my basic knowledge of `sed` :)